src/util/psbt/mod.rs

```diff
@@ -20,14 +20,14 @@
 //!
 
 use blockdata::script::Script;
-use blockdata::transaction::Transaction;
+use blockdata::transaction::{ TxOut, Transaction};
 use consensus::{encode, Encodable, Decodable};
 use consensus::encode::MAX_VEC_SIZE;
+pub use util::sighash::Prevouts;
 
 use prelude::*;
 
 use io;
-
```

Comment:
Please no 'random' whitespace changes. The reason is that it makes reviewing the PR harder because reviewers must stop to think what is this, why is it being done, is it correct, is it a step backwards in readability. All easy questions but unnecessary for getting your work merged. The easier you make it for reviewers to ack your work the easier you'll get PR's merged, that's a win for everyone :)

Reply:
Yeah, silly mistake of mine, do you guys use any sort of linter ?

Reply:
Yeah, silly mistake of mine ... will take care of it from next commits.

Reply:
Cheers mate, no stress. If I had a dollar for ever mistake I made in this repo in the last three months I'd be fiat rich :)